### DISCUSSION TITLE
Broken files after trimming

### DISCUSSION THREAD

**Discussion by ruediheimlicher** [mifi/lossless-cut#1787] [Support]
I am downloading tv films (mp4) from the internet, trimming them with LosslessCut and storing the result on a HD. I then play them  with OSMC on a Raspi3+ or Raspi4 on my tv.
It happens that there is some stuttering on the just downloaded videos. Playing them with VLC or QuicktimePlayer on my MacBookPro M1 or MacMini M1 with macOS Monterey or Ventura runs overrunning the stuttering.
With some trimmed files, with or without stuttering, there is an error at a certain moment that blocks the play with VLC or QuicktimePlayer  in a way that I have to hard reset the machine with the power button. 

Playing the vids with OSMC with my Raspi also blocks everything. Sometimes first showing "Buffering" forever

I am using LosslesCut and OSMC for many years. The problems s tarted some months ago. I tried many versions of LosslessCut without stable success.
Any ideas?

MediaInfo if a file after downloading:
[MediaInfo_orig.txt](https://github.com/mifi/lossless-cut/files/13460294/MediaInfo_orig.txt)

Video stoppes afer a few seconds, showing "Buffering" forever. 
Before further actions I have to go back to "Files" and choose another File. Sometimes I need to unplug.

MediaInfo of trimmed vid:
[MediaInfo_n.txt](https://github.com/mifi/lossless-cut/files/13460298/MediaInfo_n.txt)
Video stopps afer about 4 Minutes, no more actions possible, needs unplug of the Raspi.

**Comment by mifi** [comment]
no other ideas than trying different workarounds from https://github.com/mifi/lossless-cut/blob/master/issues.md
